internal/chartverifier/checks/helm.go

```diff
@@ -119,21 +119,18 @@ func (c *chartCache) Get(uri string) (ChartCacheItem, bool, error) {
 	}
 }
 
-func (c *chartCache) Add(uri string, repositoryCache string, chrt *chart.Chart) (ChartCacheItem, error) {
+func (c *chartCache) Add(opts *CheckOptions, chrt *chart.Chart) (ChartCacheItem, error) {
 	var (
 		err          error
 		userCacheDir string
 	)
-	if repositoryCache == "" {
-		userCacheDir, err = os.UserCacheDir()
-		if err != nil {
-			return ChartCacheItem{}, err
-		}
-	} else {
-		userCacheDir = repositoryCache
+
+	userCacheDir = getCacheDir(opts)
+	if userCacheDir == "" {
+		return ChartCacheItem{}, err
 	}
 	cacheDir := path.Join(userCacheDir, "chart-verifier")
```

Comment:
If this line `cacheDir := path.Join(userCacheDir, "chart-verifier")` required here now? getCacheDir(opts *CheckOptions) func seems to be returning the repocache/usercache with chart-verifier joined to the path.

Reply:
Goof spot, no longer needed. Removed.